mystyle.css

```diff
@@ -1,12 +1,18 @@
+body{margin:0;}
 #logo {
 float: left;
-width: 20vw;
+height:10vw;
+width:10vw;
 }
 #tytul {
 float: left;
+background-color: rgb(179, 179, 179);
```

Comment:
`rgb(179, 179, 179)` to `#b3b3b3`, a można to skrócić do `#bbb` albo `#aaa` bardziej